### PR TITLE
Chris GSOD2021

### PR DESCRIPTION
**Pull request by chris-4444**
Note: <Ongoing Progress>
________________________________________________
Please make sure to check the following boxes creating your pull request, thank you very much!

- [ ] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [ ] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [ ] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [ ] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [ ] Select a beautiful and meaningful teaser image for your blog
- [ ] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Review by huan — CHANGES_REQUESTED** [review]
I'm looking forward to seeing your PR to be ready for reviewing!

**Comment by Rohitesh-Kumar-Jain** [comment]
Hi @chris-4444 is the PR ready to be merged or you would like to push few more commits?

**Comment by chris-4444** [comment]
no more commits, since my new proposal was the joint proposal with joseph. ty @Rohitesh-Kumar-Jain

File: Chris GSOD2021 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (Rohitesh-Kumar-Jain) [inline Chris GSOD2021:1]: LGTM

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
@huan PR is ready to get merged.

**Comment by lijiarui** [comment]
Hi @shraddhavp As GSoD Project PR Review Rules in https://github.com/wechaty/wechaty.js.org/issues/1045

This PR should follow our rules and get enough approval, then I will review this PR. So I remove the `ready to merge` tag and feel free to add this tag when it gets enough approval.

**Review by lijiarui — APPROVED** [review]
Seems it's too simple..

**Review by lijiarui — CHANGES_REQUESTED** [review]
As profile request:

- [ ] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [ ] Create your contributor profile (if you are a first time contributor)
- [ ] Add avatar for your contributor profile

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

**Comment by Rohitesh-Kumar-Jain** [comment]
@chris-4444 either work upon this, or close this : )

**Comment by huan** [comment]
ping @chris-4444 

We will close this PR if there has no more commits send to it before the end of Sep.

**Comment by lijiarui** [comment]
ping @chris-4444 

We will close this PR if there has no more commits send to it before the end of Sep.

**Comment by lijiarui** [comment]
ping @chris-4444

I will close this PR tomorrow if there has no more commits send to it.

**Comment by chris-4444** [comment]
No worries, we can close this PR. I was creating my own proposal and then, came the [joint proposal](https://docs.google.com/document/d/1VxIByga8Lui6raoyrosfvjQ8zelpwk-1ci7uk8mZ57M/edit#heading=h.4ajz67fdcwye) after the Wechaty meeting, just before the final TW selection. Ty. 😎

**Pull request closed without merging**